src/store/models/CommonRole.js

```diff
@@ -0,0 +1,34 @@
+import { attr, many, Model } from 'redux-orm'
+
+// export class Responsibility extends Model { }
```

Comment:
??

Reply:
for some reason having this causes issues, cant figure it out